packages/react-native-bpk-component-calendar/src/BpkCalendar-test.common.js

```diff
@@ -157,9 +160,9 @@ const commonTests = () => {
           <BpkCalendar
             selectionType={SELECTION_TYPES.multiple}
             selectedDates={[
-              new Date(2019, 4, 19),
-              new Date(2019, 4, 21),
-              new Date(2019, 4, 29),
+              new Date(Date.UTC(2019, 4, 19)),
+              new Date(Date.UTC(2019, 4, 21)),
+              new Date(Date.UTC(2019, 4, 29)),
```

Comment:
My assumption was that this conversion to UTC would happen in the Calendar component's JS so that consumers don't need to worry about it?

Reply:
No the whole idea is that the public interface in RN should take only UTC dates as input and output. This is to align with Android